Tasks/XamarinComponentRestoreV0/task.json

```diff
@@ -2,17 +2,18 @@
     "id": "ff50fc97-da8c-4683-b014-34c15315ee5f",
     "name": "XamarinComponentRestore",
     "friendlyName": "Xamarin Component Restore",
-    "description": "Restores Xamarin components for the specified solution",
+    "description": "[Deprecated] Restores Xamarin components for the specified solution",
     "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=786653)",
     "category": "Package",
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 131,
+        "Minor": 136,
```

Comment:
You need to set deprecated to true in the JSON like here: https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/XcodePackageiOSV0/task.json#L17